intl/msg_hash_us.h

```diff
@@ -1520,6 +1576,22 @@ MSG_HASH(
    MENU_ENUM_SUBLABEL_MENU_DRIVER,
    "Menu driver to use."
    )
+MSG_HASH(
+   MENU_ENUM_LABEL_HELP_MENU_DRIVER_XMB,
+   "XMB is a RetroArch GUI that looks like a 7th generation console menu. It can support same features as Ozone."
+   )
+MSG_HASH(
+   MENU_ENUM_LABEL_HELP_MENU_DRIVER_OZONE,
+   "Ozone is the default GUI of RetroArch on most platforms. It is optimized for navigation with a game controller."
```

Comment:
Is this proper English? Kinda reads a bit like Spanish to me.

Reply:
I could swear I copied this from somewhere, but I can't find it now.

Reply:
Seems fine to me?